Code changes for integrating nvidia v5.0

anandhu-eng opened the pull request.

arjunsuresh commented on script/app-mlperf-inference-nvidia/meta.yaml:
```diff
@@ -272,7 +272,9 @@ deps:
+    names:
+    - pycuda
+    version: "2023.1"
 
```
Restrict it to v5. 0?

arjunsuresh commented on script/build-mlperf-inference-server-nvidia/meta.yaml:
```diff
@@ -111,7 +111,7 @@ deps:
   # Detect pycuda
   - tags: get,generic-python-lib,_pycuda
-    version: "2022.2.2"
+    version: "2023.1"
```
Does this work for Nvidia v4.0?

arjunsuresh commented on script/run-mlperf-inference-app/meta.yaml:
```diff
@@ -366,7 +366,6 @@ variations:
     group: benchmark-version
   
   r5.0-dev:
-    default: true
```
This should be the default for now right?

anandhu-eng commented:
Hi @arjunsuresh , I have cleaned the code in latest PR. I have reverted the PyCuda version as it is not tested for 4.0. I can make changes if the upgraded PyCuda version does not fail the workflow for 4.0.
